docs/src/13_annex.adoc

```diff
@@ -0,0 +1,80 @@
+ifndef::imagesdir[:imagesdir: ../images]
+
+[[section-annex]]
+== Annex
+=== Functional Requirements
+==== Users Sign up.
+[none or no-bullet]
+* FR-USU 1. The system will offer to an unregistered user a registration form.
+* FR-USU 2. The system will request the necessary data to register the new user.
+[none or no-bullet]
+** FR-USU 2.1. The system will request the following data to register the new user:
+[none or no-bullet]
+*** FR-USU 2.1.1. Username.
+*** FR-USU 2.1.2. Email address.
+*** FR-USU 2.1.3. Password.
+*** FR-USU 2.1.4. All data fields are mandatory.
+* FR-USU 3. When any value entered by the new user is invalid, the user will receive a message notifying the error and will not be able to register.
+* FR-USU 4. When all the values entered by the new user are valid, the system will check if the user is already registered in the persistence system.
+[none or no-bullet]
+** FR-USU 4.1. When a user with the same data is already found in the persistence system, the user will not be able to create a new account since it already exists.
+** FR-USU 4.2. When no user matching the data is found in the persistence system.
+[none or no-bullet]
+*** FR-USU 4.2.1. The new user will be registered in the system.
+
+==== Users Log in.
+[none or no-bullet]
+* FR-ULI 1. The system must allow an unidentified user to log in.
+[none or no-bullet]
+** FR-ULI 1.1. The system will request the email address as the user identifier.
+[none or no-bullet]
+*** FR-ULI 1.1.1. The system must check that its format is valid.
+*** FR-ULI 1.1.2. It is a mandatory field.
+** FR-ULI 1.2. The system will request the user's password.
+[none or no-bullet]
+*** FR-ULI 1.2.1. It is a mandatory field.
+** FR-ULI 1.3. The system will automatically validate the entered data to verify when it corresponds to a registered user account.
+[none or no-bullet]
+*** FR-ULI 1.3.1. When the user is not stored in the persistence system, an error message will be displayed.
+*** FR-ULI 1.3.2. When the user exists in the persistence system, but the passwords do not match, a message will be displayed to the user notifying them of the error.
+*** FR-ULI 1.3.3. When the user is stored in the persistence system and the passwords match, the user will be logged in.
```

Comment:
Please, reformulate as "the hashes of the passwords (do not) match" as this can lead to misunderstandings.

Reply:
That's design dependent.